netsuite/10.0/modules/ROOT/pages/netsuite-troubleshooting.adoc

```diff
@@ -69,6 +69,9 @@ To log requests and responses when using the connector, add this XML statement t
 
 To avoid receiving a permissions error when fetching metadata, enable the `Control SuiteScript and Workflow Triggers in Web Services Request` option in NetSuite for the role used in the connector configuration.
 
+You will also need to enable the `Custom Segments` option in NetSuite to be able to fetch metadata with custom fields.
```

Comment:
Try not to use future tense if you don't have to. (One of the tech writing rules - not sure why - except that it makes the text shorter.)

You also need to enable...

Reply:
Good catch, thanks Diane.